layers/+tools/finance/packages.el

```diff
@@ -43,6 +43,12 @@
          "t" 'ledger-insert-effective-date
          "y" 'ledger-set-year
          "RET" 'ledger-set-month)
+      (spacemacs/set-leader-keys-for-major-mode 'ledger-reconcile-mode
+        (or dotspacemacs-major-mode-leader-key ",") 'ledger-reconcile-toggle
```

Comment:
I am not sure what you are doing here 😸 Since `dotspacemacs-major-mode-leader-key` is declared via `defvar` and it's default value is `","` I can't understand why the check is needed. You know, if someone sets `dotspacemacs-major-mode-leader-key` to `nil`, then it's not our problem 😸